.github/workflows/ci-project.yml

```
@@ -0,0 +1,51 @@
+name: Test setup 
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches: 
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Install node required 
+      uses: actions/setup-node@v3
+      with:
+        node-version: 18.x
+
+    - name: Ruby setup
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: '3.3.0' 
+         
+    - name: Install dependencies
```

Please Mention clearly the dependencies you are installing.
```suggestion
    - name: Install Ruby dependencies
```